addons/payment/migrations/13.0.1.0/post-migration.py

```diff
@@ -0,0 +1,23 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
+
+unlink_by_xmlid = [
+    'payment.payment_acquirer_custom'
+]
+
+
+def map_payment_acquirer_state(cr):
+    openupgrade.map_values(
```

Comment:
don't use map_values in this case, because I think it's a conditional case: map 'prod' option to 'enabled' if 'website_published = True , else map 'prod' to 'disabled'

Reply:
done